### ISSUE TITLE
Release client-side independently? 

### ISSUE THREAD

**Issue by joelkuiper** [nlplab/brat#1156]
It would be nice if the client-side highlighting tool was packaged and documented independently (preferably with a documented API) so one can integrate the beautiful highlighting tools for their own purposes easily without the whole framework.

**Comment by spyysalo** [comment]
Thanks for the suggestion! https://spyysalo.github.io/annodoc/ uses the client-side independently (with a bit of glue), maybe this can help?

**Comment by joelkuiper** [comment]
Definitely looks interesting! I'll take a look. 
From the looks of it you can pass in a block of .ann files (or related) and it'll highlight it correctly. 
Does the .ann format also support having more complex data shown on-hover (I know not a primary use case, but definitely interesting)?

**Comment by spyysalo** [comment]
> Does the .ann format also support having more complex data shown on-hover

Yep, the current implementation shows attribute values and free-text comments attached to each annotation on hover.
